code/_onclick/hud/new_player.dm

```diff
@@ -256,13 +280,56 @@
 	set_button_status(TRUE)
 	UnregisterSignal(SSticker, COMSIG_TICKER_ENTER_PREGAME)
 
-/atom/movable/screen/lobby/button/settings
+//Subtype the bottom buttons away so the collapse/expand shutter goes behind them
+/atom/movable/screen/lobby/button/bottom
+	layer = LOBBY_BOTTOM_BUTTON_LAYER
 	icon = 'icons/hud/lobby/bottom_buttons.dmi'
+	always_shown = TRUE
+
+/atom/movable/screen/lobby/button/bottom/Initialize(mapload)
+	. = ..()
+	if(hud)
+		RegisterSignal(hud, COMSIG_HUD_LOBBY_COLLAPSED, PROC_REF(collapse_button))
+		RegisterSignal(hud, COMSIG_HUD_LOBBY_EXPANDED, PROC_REF(expand_button))
```

Comment:
if you used signals to hide, then you'd need to override the receiving procs here, rather then re-registering

Reply:
we ball
https://github.com/tgstation/tgstation/pull/76443/commits/7d1dbbcb8e604ae8d176707862cb563c7b32674f